src/main/java/uk/gov/companieshouse/client/PaymentsClient.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.client;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import uk.gov.companieshouse.config.PaymentsConfig;
+import uk.gov.companieshouse.exception.NotFoundException;
+import uk.gov.companieshouse.model.dto.payment.RefundResponse;
+
+import java.util.Optional;
+
+@Service
+public class PaymentsClient {
+
+    private static final String REFUNDS_URI = "/payments/{paymentId}/refunds";
+    private static final double REFUND_AMOUNT = 8.00;
+
+    private final PaymentsConfig config;
+
+    @Autowired
+    public PaymentsClient(PaymentsConfig config) {
+        this.config = config;
+    }
+
+    public Optional<RefundResponse> refundPayment(String paymentId) {
+        try {
+            return Optional
```

Comment:
Content-Type, Authorization and Accept headers are all missing. Check DocumentRenderClient and CompanyOfficersClient for examples (ignore assetId and templateName headers in DocumentRender)